### PR TITLE
Fix some clang build warnings

### PR DESCRIPTION
**Pull request by kisielk**

### CODE REVIEW

File: source/DEINDUGens/GreyholeRaw.cpp [hunk 1]
```diff
@@ -1119,7 +1119,7 @@ static std::string normalizeClassName(const std::string& name)
 
   unsigned int i=0;
   bool upnext=true;
-  while (c=name[i++]) {
+  while ((c=name[i++])) {
```
Comment (snappizz) [inline source/DEINDUGens/GreyholeRaw.cpp]: This file is generated by FAUST. It's not a good idea to edit generated code.

**Comment by kisielk** [comment]
Removed changes to DEINDUGens

**Comment by bagong** [comment]
Fine on Windows!

**Comment by brianlheim** [comment]
Sorry this took so long to merge!

**Pull request merged**